### PR TITLE
Updating to bittensor 7.0

### PR DESCRIPTION
**Pull request by renesweet24**

### CODE REVIEW

File: neurons/validators/utils/uids.py [hunk 1]
```diff
@@ -52,41 +54,40 @@ def get_top_miner_uids(
...
+    for uid in miner_healthy_uids:
         if metagraph.axons[uid].ip not in ips:
             ips.append(metagraph.axons[uid].ip)
-            filtered_uids.append(uid)
+            miner_ip_filtered_uids.append(uid)
+
+    # Consider both of incentive and trust score
+    values = [(uid, metagraph.I[uid] * metagraph.trust[uid]) for uid in miner_ip_filtered_uids]
+
+    # # Consider only incentive
```
Comment (nnboskovic) [inline neurons/validators/utils/uids.py:78]: Does any of this code still apply? Should it be commented or removed?

File: neurons/validators/utils/uids.py [hunk 2]
```diff
@@ -52,41 +54,40 @@ def get_top_miner_uids(
...
-
-        miner_healthy_uids, _ = await ping_uids(dendrite, metagraph, miner_candidate_uids)
-
-        ips = []
-        miner_ip_filtered_uids = []
-        for uid in miner_healthy_uids:
-            if metagraph.axons[uid].ip not in ips:
-                ips.append(metagraph.axons[uid].ip)
-                miner_ip_filtered_uids.append(uid)
+    miner_candidate_uids = []
```
Comment (nnboskovic) [inline neurons/validators/utils/uids.py:57]: Was the try-catch block removed for a reason?

File: neurons/validators/utils/uids.py (hunk 1 above)
Comment (renesweet24) [reply neurons/validators/utils/uids.py:78]: it will be removed soon

Comment (renesweet24) [reply neurons/validators/utils/uids.py]: removed

**Pull request merged**